### ISSUE TITLE
"Open with P4V" creates a new P4V instance every time.

### ISSUE THREAD

**Issue by arycama** [SimpsonGSD/P4EditVS#36]
I think it would be better if this uses the existing instance of P4V instead of opening a new one. This could be a configurable option if needed, though I think making it use the existing instance by default would be best.

**Comment by SimpsonGSD** [comment]
I agree, it would be better to not keep opening new instances but re-use the first one opened.

**Comment by SimpsonGSD** [comment]
Note: If you have P4V 2024.1 and enable the `P4V 2024.1 Open in P4V Support` option in Visual Studio settings, this should just work now.

**Comment by Skylark13** [comment]
Asking here but if you prefer a new issue I can do that too:
Is there a plan to detect the installed version of p4v instead of relying on the user to enable an option? Because no matter what the option defaults to, some people will have it wrong for their version of p4v, will have problems and come looking in the github issues to see if it's already reported, and will open a new issue if not, only for you to tell them to change the setting. (not saying I just did this which led me here... But also not saying I didn't!)

**Comment by SimpsonGSD** [comment]
The previous fix was a quick fix to unblock people. It might be something to try. I've not had any reports of this issue until now so I've not revisited it since.

**Comment by SimpsonGSD** [comment]
If you can make a new issue that would be good 👍
